facade/host.go

```diff
@@ -351,6 +363,46 @@ func (f *Facade) FindReadHostsInPool(ctx datastore.Context, poolID string) ([]ho
 	return toReadHosts(hosts), nil
 }
 
+// GetHostStatuses returns the memory usage and whether or not a host is active
+func (f *Facade) GetHostStatuses(ctx datastore.Context, hostIDs []string, since time.Time) ([]host.HostStatus, error) {
+	if hostIDs == nil {
+		return []host.HostStatus{}, nil
+	}
+
+	statuses := []host.HostStatus{}
+	for _, id := range hostIDs {
+		h, err := f.GetHost(ctx, id)
+		if err != nil {
+			continue
+		}
+
+		status := host.HostStatus{HostID: id, MemoryUsage: service.Usage{}}
+		active, err := f.zzk.IsHostActive(h.PoolID, h.ID)
+		if err != nil {
+			continue
+		}
+		status.Active = active
+
+		authed, _ := f.hostRegistry.IsExpired(h.ID)
```

Comment:
I made a separate PR for this egregious error https://github.com/control-center/serviced/pull/3080